src/frontends/pytorch/src/op/device.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "pt_framework_node.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_device(NodeContext& context) {
+    auto decoder = context.get_decoder();
+    auto node = std::make_shared<PtFrameworkNode>(context.get_decoder(), context.inputs());
+    auto attrs = node->get_attrs();
+    auto device = decoder->get_device();
+    attrs["string_value"] = device;
```

Comment:
```suggestion
    attrs["string_value"] = decoder->get_device();
```
...and remove the line above

Reply:
Done